src/services/CertificateGenerationService.ts

```diff
@@ -563,6 +564,50 @@ class CertificateGenerationService {
 
         return defectString;
     }
+    /**
+     * To fetch trailer registration
+     * @param vin The vin of the trailer
+     * @param make The make of the trailer
+     * @returns A payload containing the TRN of the trailer.
+     */
+     public async getTrailerRegistration(vin: string, make: string) {
+        const config: IInvokeConfig = this.config.getInvokeConfig();
+        const invokeParams: any = {
+            FunctionName: config.functions.trailerRegistration.name,
+            InvocationType: "RequestResponse",
+            LogType: "Tail",
+            Payload: JSON.stringify({
+                httpMethod: "GET",
+                path: `${Configuration.getInstance().getTrailerServiceVersion()}/trailers/${vin}?make=${make}`,
```

Comment:
Do we need a new Class for that?
I was wondering if we could just use the existing `src/config/config.yml` and add more elements (with a new property) in there as part of its config so we don't have 2 places for the config?

I think all it needs is `/v1/trailers/<vin>` ?

Reply:
left a comment related to that in the Configuration class but for me this path should be retrieved from the event in the handler (so perhaps we pass an additional argument to the function which has the event), we shouldn't have to maintain the version number from another service into this current service